examples/pylab_examples/boxplot_demo2.py

```diff
@@ -16,6 +16,7 @@
 randomDists = ['Normal(1,1)', ' Lognormal(1,1)', 'Exp(1)', 'Gumbel(6,4)',
                'Triangular(2,9,11)']
 N = 500
+np.random.seed(0)
```

Comment:
I would suggest adding a comment that the random seeding is only to make the example reproducible